pyuvsim/tests/test_simsetup.py

```diff
@@ -677,7 +677,18 @@ def test_param_reader_errors(subdict, error, msg):
 
 
 @pytest.mark.filterwarnings("ignore:Cannot check consistency of a string-mode BeamList")
-def test_tele_parser():
+@pytest.mark.parametrize(
+    ("world", "selenoid"),
+    [
+        (None, None),
+        ("moon", "SPHERE"),
+        ("moon", "GSFC"),
+        ("moon", "GRAIL23"),
+        ("moon", "CE-1-LAM-GEO"),
+        ("moon", None),
+    ]
+)
```

Comment:
shouldn't we cover the world = "earth" case too?

Reply:
sure. I think None is equivalent to "earth" but we should make sure of that.